package.json

```diff
@@ -7,8 +7,8 @@
     "phplint": "bin/php-lint.sh",
     "phpcs": "vendor/bin/phpcs",
     "phpcs:errors": "vendor/bin/phpcs --report=bin/phpcs-eslint-report.php | eslines --processors enforce --quiet",
-    "phpcs:changed:remote": "vendor/bin/phpcs --report=bin/phpcs-eslint-report.php | eslines --quiet",
-    "phpcs:changed:local": "vendor/bin/phpcs --report=bin/phpcs-eslint-report.php | eslines --quiet -d index",
+    "phpcs:changed:remote": "vendor/bin/phpcs --cache --report=bin/phpcs-eslint-report.php | eslines --quiet",
+    "phpcs:changed:local": "vendor/bin/phpcs --cache --report=bin/phpcs-eslint-report.php | eslines --quiet -d index",
```

Comment:
Improve developer experience by caching the results between runs (the cache file will by default be created in `/tmp`). When using the cache, `phpcs` will check only the modified files instead of the entire codebase, which saves a lot of time.